locator_ros_bridge: 2.1.0-1 in 'rolling/distribution.yaml' [bloom]

slaible opened the pull request:
Increasing version of package(s) in repository `locator_ros_bridge` to `2.1.0-1`:

- upstream repository: https://github.com/boschglobal/locator_ros_bridge.git
- release repository: https://github.com/ros2-gbp/locator_ros_bridge-release.git
- distro file: `rolling/distribution.yaml`
- bloom version: `0.10.7`
- previous version for package: `null`

## bosch_locator_bridge

```
* initial version
* Contributors: Stefan Laible
```

slaible committed:
locator_ros_bridge: 2.1.0-1 in 'rolling/distribution.yaml' [bloom]

hidmic commented on rolling/distribution.yaml:
```diff
@@ -1177,6 +1177,23 @@ repositories:
+    source:
+      type: git
+      url: https://github.com/boschglobal/locator_ros_bridge.git
+      version: rolling
```
@slaible I see no `rolling` branch in the source repository.

hidmic replied:
There's a 2.1.0 tag in the `main` branch, so maybe you meant to use `main`?

slaible replied:
@hidmic Yes, the rolling version is in the main branch. Is this a problem?

wjwwood replied:
Yeah, the value here needs to be a tag or branch in the upstream repository. Which appears to not be the case for 'rolling'?

slaible replied:
@wjwwood Ok, so what do I have to do? Is it sufficient if I change version to main in line 1195 of distribution.yaml? (The main branch contains the rolling version.)

tfoote replied:
Yes, if you update 1195 as well as 1184 to be `main` that should let it pass CI.

wjwwood replied:
That is not covered in the tutorial.

The tool prompts you, asking if you want to have a source and/or doc entry. At that point you put in the repository url, vcs type (e.g. `git`), and the version. I think you just misunderstood that `version` in that context means a tree-ish or commit like thing, either a branch or tag or commit hash for git. Basically you're telling us where to get the source code for documentation or building from source. This might be different for each one case. This version is not for "which ros distribution", as we e already know it's for rolling at this point.

slaible replied:
Ah ok, thank you for the explanation!

slaible committed:
Update distribution.yaml

Change version to be main for the rolling version of package bosch_locator_bridge
